admin_app/src/app/content/page.tsx

```diff
@@ -54,8 +54,10 @@ const CardsPage = () => {
 
   React.useEffect(() => {
     const fetchTags = async () => {
-      const data = await apiCalls.getTagList(token!);
-      setTags(data);
+      if (token) {
```

Comment:
Should we add `token` to line 64 so this runs once the token has been fetched?

Reply:
I think we need both! But please do try if it works with just adding to line 64

Reply:
You are right we need both.